scripts/components/profiles/line.component.js

```diff
@@ -36,87 +39,142 @@ export default class {
       .append('g')
       .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
+    let tempArr = data.lines.map(line => line.values);
+    let allYValues = [];
+    for(let i = 0; i < tempArr.length; i++) {
+      allYValues = [...allYValues, ...tempArr[i]];
+    }
```

Comment:
In this case, `[].concat(data.lines.map(line => line.values))` return the same as `data.lines.map(line => line.values)`.
But you have help me to do this `[].concat.apply([], data.lines.map(line => line.values))` 😄